Readme.md

```diff
@@ -15,7 +15,7 @@
 >>> age = 54
 
 >>> [fmt|Person's name is {name}, age is {age:x}|]
-"Person's name is Dave, age is 36"
+"Person's name is Dave, age is 54"
```

Comment:
This is wrong. 54 in hexa is 36.

Reply:
ohh.... there is an hexa modificator in the very first example, without hexa prefix, before it's even mentioned...
ok, I'll fix the pattern then